lib/sqlalchemy/testing/suite/test_reflection.py

```diff
@@ -292,8 +292,11 @@ def define_reflected_tables(cls, metadata, schema):
                     Column("q", sa.String(5)),
                     test_needs_fk=True,
                 )
-                Index("noncol_idx_nopk", noncol_idx_test_nopk.c.q.desc())
-                Index("noncol_idx_pk", noncol_idx_test_pk.c.q.desc())
+
+                if testing.requires.indexes_with_ascdesc.enabled:
```

Comment:
we have to add the exlusion also to:
```

    @testing.requires.index_reflection
    def test_get_noncol_index_no_pk(self):
        self._test_get_noncol_index("noncol_idx_test_nopk", "noncol_idx_nopk")

    @testing.requires.index_reflection
    def test_get_noncol_index_pk(self):
        self._test_get_noncol_index("noncol_idx_test_pk", "noncol_idx_pk")

```

right?

Reply:
Good catch, thanks.